CONTRIBUTING.md

```diff
@@ -86,3 +86,41 @@ described below:
   "proposal finalized" label.  This indicates that it is ready to be
   implemented by a platform developer, see the [process](#contributing) above.
 - Once a Pull Request is approved, it will be merged into the 'master' branch.
+
+## Spec Release Process
+
+Any member of the OSBAPI PMC can request that the accepted spec changes on
+master are released in a new version of the spec. They will do this by
+creating a new PR with the tile of the proposed release. For example,
+"Release Proposal: v2.20".
+
+### Prepare a PR
+
```

Comment:
Another step needs to be added to the *Prepare a PR* section. The `spec.md` has a *Changes Since v*.*` section which also needs to be updated.